packages/trader/src/Modules/Trading/Components/Form/ContractType/contract-type-list.jsx

```diff
@@ -7,31 +6,28 @@ const ContractTypeList = ({
     handleInfoClick,
     handleSelect,
     is_equal,
-    list,
+    list = [],
     name,
     value,
-}) =>
-    (
-        Object.keys(list).map(key => (
-            // TODO: Remove this line after other contracts are ready to be served
-            !['In/Out', 'Asians'].includes(key) &&
-            <React.Fragment key={key}>
-                <div className='contract-type-list'>
-                    <div className='contract-type-list__label'><span>{key}</span></div>
-                    <div className='contract-type-list__contracts-wrapper'>
-                        <ContractTypeItem
-                            contracts={list[key]}
-                            name={name}
-                            value={value}
-                            handleSelect={handleSelect}
-                            handleInfoClick={handleInfoClick}
-                            is_equal={is_equal}
-                        />
-                    </div>
-                </div>
-            </React.Fragment>
-        ))
-    );
+}) => (
+    Object.keys(list).map(key => (
```

Comment:
why do we need Object.keys(list) here when were accessing the list item below with `list[key]`?  shouldn't list.map() be enough?